src/gui/entry/EditEntryWidget.h

```diff
@@ -122,14 +144,20 @@ private slots:
 
     bool m_create;
     bool m_history;
+#ifdef WITH_XC_SSHAGENT
+    bool m_sshAgent;
```

Comment:
Suggest renaming this to ```m_sshAgentEnabled``` as it is named now it implies holding an object not a boolean.